mantidproject/mantid
# workbench setting documentation

Added documentation on the workbench settings
Added help button to workbench settings menu that opens help window.

**To test:**
1. build docs-html and docs-qthelp
1. open workbench
1. open file --> settings
1. click on the ? button
1. check that the help window opens on the Settings page and that the documentation is suitable.

Fixes #26135 

*This does not require release notes* because **It is an addition to documentation**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
